autosynth failed PR log entry all on one line

Looking at this issue:
https://github.com/googleapis/nodejs-dataproc/issues/386

The inline log in the PR body is all on a single line, making it very hard to read.

We should have seen this log on the first link we clicked:
```
> @google-cloud/asset@3.1.0 compile /usr/local/google/home/rennie/gitrepos/nodejs-asset
> tsc -p . && cp -r protos build/

node_modules/google-gax/build/protos/iam_service.d.ts:4291:40 - error TS2304: Cannot find name 'Long'.

4291             positiveIntValue?: (number|Long|null);
                                            ~~~~

node_modules/google-gax/build/protos/iam_service.d.ts:4294:40 - error TS2304: Cannot find name 'Long'.

4294             negativeIntValue?: (number|Long|null);
                                            ~~~~

node_modules/google-gax/build/protos/iam_service.d.ts:4322:46 - error TS2304: Cannot find name 'Long'.

4322             public positiveIntValue: (number|Long);
                                                  ~~~~

node_modules/google-gax/build/protos/iam_service.d.ts:4325:46 - error TS2304: Cannot find name 'Long'.

4325             public negativeIntValue: (number|Long);
                                                  ~~~~

node_modules/google-gax/build/protos/operations.d.ts:3762:40 - error TS2304: Cannot find name 'Long'.

3762             positiveIntValue?: (number|Long|null);
                                            ~~~~

node_modules/google-gax/build/protos/operations.d.ts:3765:40 - error TS2304: Cannot find name 'Long'.

3765             negativeIntValue?: (number|Long|null);
                                            ~~~~

node_modules/google-gax/build/protos/operations.d.ts:3793:46 - error TS2304: Cannot find name 'Long'.

3793             public positiveIntValue: (number|Long);
                                                  ~~~~

node_modules/google-gax/build/protos/operations.d.ts:3796:46 - error TS2304: Cannot find name 'Long'.

3796             public negativeIntValue: (number|Long);
                                                  ~~~~

node_modules/google-gax/build/protos/operations.d.ts:4485:31 - error TS2304: Cannot find name 'Long'.

4485             seconds?: (number|Long|null);
                                   ~~~~

node_modules/google-gax/build/protos/operations.d.ts:4501:37 - error TS2304: Cannot find name 'Long'.

4501             public seconds: (number|Long);
                                         ~~~~

protos/protos.d.ts:15:23 - error TS2307: Cannot find module 'long' or its corresponding type declarations.

15 import * as Long from "long";
                         ~~~~~~

protos/protos.d.ts:16:28 - error TS2307: Cannot find module 'protobufjs' or its corresponding type declarations.

16 import * as $protobuf from "protobufjs";
                              ~~~~~~~~~~~~


Found 12 errors.
```

--- a/autosynth/multi.py
+++ b/autosynth/multi.py
@@ -93,7 +93,7 @@ def synthesize_library(
         logger.error(f"Synthesis failed for {library['name']}")
     return {
         "name": library["name"],
-        "output": output,
+        "output": output.decode("utf-8"),
         "error": error,
         "skipped": skipped,
     }

--- a/tests/test_multi.py
+++ b/tests/test_multi.py
@@ -42,7 +42,7 @@ def test_synthesize_library_success():
         )
 
     assert result["name"] == "test1"
-    assert result["output"] == b"success"
+    assert result["output"] == "success"
     assert result["error"] is False
     assert result["skipped"] is False
 
@@ -63,7 +63,7 @@ def test_synthesize_library_failure():
         )
 
     assert result["name"] == "test1"
-    assert result["output"] == b"something failed"
+    assert result["output"] == "something failed"
     assert result["error"] is True
     assert result["skipped"] is False
 
@@ -84,7 +84,7 @@ def test_synthesize_library_skip():
         )
 
     assert result["name"] == "test1"
-    assert result["output"] == b"nothing changed"
+    assert result["output"] == "nothing changed"
     assert result["error"] is False
     assert result["skipped"] is True
 
@@ -95,19 +95,19 @@ def test_make_report():
         [
             {
                 "name": "test1",
-                "output": b"some output data",
+                "output": "some output data",
                 "error": False,
                 "skipped": False,
             },
             {
                 "name": "test2",
-                "output": b"something failed",
+                "output": "something failed",
                 "error": True,
                 "skipped": False,
             },
             {
                 "name": "test3",
-                "output": b"something skipped",
+                "output": "something skipped",
                 "error": False,
                 "skipped": True,
             },

--- a/tests/testdata/sponge-log-golden.xml
+++ b/tests/testdata/sponge-log-golden.xml
@@ -4,7 +4,7 @@
     <testsuite name="test1" tests="1" errors="0" failures="0" skipped="0">
         <testcase classname="test1" name="synthesize">
             
-            <system-out>b&#39;some output data&#39;</system-out>
+            <system-out>some output data</system-out>
             
         </testcase>
     </testsuite>
@@ -12,7 +12,7 @@
     <testsuite name="test2" tests="1" errors="0" failures="1" skipped="0">
         <testcase classname="test2" name="synthesize">
             
-            <failure>b&#39;something failed&#39;</failure>
+            <failure>something failed</failure>
             
         </testcase>
     </testsuite>
@@ -20,7 +20,7 @@
     <testsuite name="test3" tests="1" errors="0" failures="0" skipped="1">
         <testcase classname="test3" name="synthesize">
             
-            <system-out>b&#39;something skipped&#39;</system-out>
+            <system-out>something skipped</system-out>
             
         </testcase>
     </testsuite>

